### PR TITLE
Extend README

### PR DESCRIPTION
**Pull request by jdemaeyer**
Once we're happy with this I'll submit a PR to update the [doc page on scrapinghub.com](http://doc.scrapinghub.com/shub.html) (leaving out the advanged config section) and edit [this knowledge base entry](http://support.scrapinghub.com/topic/205471-deploying-your-project-to-scrapy-cloud/) (to the quickstart section?)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -82,21 +82,180 @@ folder as ``scrapy.cfg``. However, ``shub`` will try all parent directories
...
+    Successfully downloaded loginform
+    Package fetched successfully
+    Uncompressing: loginform-1.0.tar.gz
+    Building egg in: /tmp/shub/loginform-1.0
+    Deploying dependency to Scrapy Cloud project "12345"
+    {"status": "ok", "egg": {"version": "loginform-1.0", "name": "loginform"}}
+    Deployed eggs list at: https://dash.scrapinghub.com/p/12345/eggs
+
+
+Scheduling and retrieving jobs
```
Comment (pablohoffman) [inline README.rst]: Scheduling jobs and fetching job data?

Comment (jdemaeyer) [inline README.rst]: Better! thanks

File: README.rst [hunk 2]
```diff
@@ -82,21 +82,180 @@ folder as ``scrapy.cfg``. However, ``shub`` will try all parent directories
...
+    $ shub requests 1/1/1
+    {"status": 200, "fp": "1ff11f1543809f1dbd714e3501d8f460b92a7a95", "rs": 138137, "_key": "1/1/1/0", "url": "http://blog.scrapinghub.com", "time": 1449834387621, "duration": 238, "method": "GET"}
+    {"status": 200, "fp": "418a0964a93e139166dbf9b33575f10f31f17a1", "rs": 138137, "_key": "1/1/1/0", "url": "http://blog.scrapinghub.com", "time": 1449834390881, "duration": 163, "method": "GET"}
+
+
+Advanced Configuration
+----------------------
+
+Besides projects you can configure different endpoints for project
+deployments::
```
Comment (pablohoffman) [inline README.rst]: not just for deployments, right?. don't other commands (items, logs) work with extra endpoints?

Comment (jdemaeyer) [inline README.rst]: `schedule` does, but job resource fetching does not so far (#94). I'd say drop "for project deployments" and fix #94 soon

File: README.rst [hunk 3]
```diff
@@ -82,21 +82,180 @@ folder as ``scrapy.cfg``. However, ``shub`` will try all parent directories
...
+    {"name": "Another product", description": "Another description"} 
+
+::
+
+    $ shub requests 1/1/1
+    {"status": 200, "fp": "1ff11f1543809f1dbd714e3501d8f460b92a7a95", "rs": 138137, "_key": "1/1/1/0", "url": "http://blog.scrapinghub.com", "time": 1449834387621, "duration": 238, "method": "GET"}
+    {"status": 200, "fp": "418a0964a93e139166dbf9b33575f10f31f17a1", "rs": 138137, "_key": "1/1/1/0", "url": "http://blog.scrapinghub.com", "time": 1449834390881, "duration": 163, "method": "GET"}
+
+
+Advanced Configuration
```
Comment (pablohoffman) [inline README.rst]: I'd highlight that this section is only useful for Scrapinghub staff/developers, or remove it altogether.

Comment (pablohoffman) [inline README.rst]: How about:

API endpoints (only for Scrapinghub staff)

**Pull request merged**